tests/src/e2e.spec.ts

```diff
@@ -1,51 +1,95 @@
 import type { BrowserWindow } from 'electron';
 import type { ElectronApplication, JSHandle, Page } from 'playwright';
 import { _electron as electron } from 'playwright';
-import { afterAll, beforeAll, expect, test } from 'vitest';
+import { afterAll, beforeAll, expect, test, describe } from 'vitest';
 import { expect as playExpect } from '@playwright/test';
-import { existsSync } from 'node:fs';
+import { existsSync, copyFileSync, renameSync } from 'node:fs';
+import { execSync } from 'node:child_process';
 import { rm } from 'node:fs/promises';
+import * as os from 'node:os';
+import * as path from 'node:path';
 
 let electronApp: ElectronApplication;
 
 let page: Page;
-
-beforeAll(async () => {
-  // remove all videos/screenshots
-  if (existsSync('tests/output')) {
-    console.log('Cleaning up output folder...');
-    await rm('tests/output', { recursive: true, force: true });
-  }
-
-  electronApp = await electron.launch({
-    args: ['.'],
-    recordVideo: {
-      dir: 'tests/output/videos',
-      size: {
-        width: 1050,
-        height: 700,
+const userHome = os.homedir();
+const navBarItems = ['Dashboard', 'Containers', 'Images', 'Pods', 'Volumes', 'Settings'];
+const settingsPath = path.join(
+  userHome,
+  '.local',
+  'share',
+  'containers',
+  'podman-desktop',
+  'configuration',
+  'settings.json',
+);
+const settingsBackupPath = path.join(
+  userHome,
+  '.local',
+  'share',
+  'containers',
+  'podman-desktop',
+  'configuration',
+  'settings_backup.json',
+);
+const kindPath = path.join(
+  userHome,
+  '.local',
+  'share',
+  'containers',
+  'podman-desktop',
+  'extensions-storage',
+  'podman-desktop.kind',
+  'kind-linux-amd64', // ToDo: change according to OS
+);
+
+describe('Basic e2e verification of podman desktop', async () => {
```

Comment:
Hello, why move the global beforeAll/afterAll into a describe ?
I don't get the rational here ?

Reply:
You are right, this will be global before/after all for all test cases/suites. Specialized before/after(all) can be in logical test blocks.